interpreter/cling/tools/packaging/cpt.py

```diff
@@ -2375,7 +2383,8 @@ def custom_input(prompt, always_yes=False):
         tarball()
         cleanup()
 
-if args['tarball_tag']:
+if bool(args['tarball_tag']) ^ bool(args['tarball_tag_build']):
```

Comment:
Can `^` be replaced by `or`?

Reply:
I wanted to use an exclusive or so that two of the options couldn't be called at the same time

Reply:
I would prefer to use or and make this check before we even come here. We can check and diagnose this condition where the user will know what they did wrong…

Reply:
Hi @vgvassilev just added the changes like you said